change/@fluentui-react-file-type-icons-39aceee7-fe31-4d84-9f6b-bac0108d5d72.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33167&commitId=6712cc28204a81b62b466f704ca908e36da5dce6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Events.chromium.png|2| Changed|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
some noise false positives appeared in the charting package and the tooltip control, which are unrelated to the album entry for filetype icon